typescript/pages.ts

```diff
@@ -0,0 +1,109 @@
+import { Component } from "../typescript/component";
+import { Image } from "../typescript/action";
+import { Entry, HeaderProps ,FooterProps } from "./layout";
+
+type Object = {
```

Comment:
I think its better not to use Object as type. It will create conflict with the native Object.
Rename it to something more relavant